packages/contracts/contracts/optimistic-ethereum/libraries/standards/IWETH9NoERC20.sol

```diff
@@ -0,0 +1,14 @@
+// SPDX-License-Identifier: GPL-2.0-or-later
+pragma solidity =0.7.6;
+
+/// @title Interface for WETH9's non-ERC20 functions
+interface IWETH9NoERC20 {
```

Comment:
I just noticed that the standard `IWETH9` actuall `is IERC20` where the `IERC20` is OZ, same as our OVM_ETH.  Can we just inherit the same way?  Seems very slightly cleaner.